### DISCUSSION TITLE
Documentation didn't show up, how do I do next?

### DISCUSSION THREAD

**Discussion by wei18** [SwiftPackageIndex/SwiftPackageIndex-Server#2867] [Q&A]
**Please describe the bug**

I modified [.spi.yml@af70860](https://github.com/Wei18/github-rest-api-swift-openapi/blob/main/.spi.yml) and verified on [validate-spi-manifest](https://swiftpackageindex.com/validate-spi-manifest), wait for days but not working.

The only first time added [.spi.yml@a97557c](https://github.com/Wei18/github-rest-api-swift-openapi/blob/a97557c42628b2621faf3f8e8c3fa8e1fcaa2b93/.spi.yml) worked.
(doc: https://swiftpackageindex.com/Wei18/github-rest-api-swift-openapi/main/documentation/githubrestapiissues)

Thanks

**Comment by finestructure** [comment]
Thanks for your report, @Wei18 , I'm looking into this.

**Reply by finestructure** [reply]
OK, the problem is that your `.spi.yml` file is too large. We're currently imposing a limit of 1_000 bytes and your file is 1103 bytes. I'll lift our limit a bit but this will take a moment to be available. In the meantime if you limit the number of documentation targets it should work.

Please note that we are also imposing a 10 minute limit on our builders to build a package and generate documentation. I haven't look at the build time for your package but given the large number of documentation targets, there's a chance this might be a problem.

**Reply by wei18** [reply]
Thank you for your help. Okay, I understand. I will try other ways to reduce the number of targets.

**Comment by daveverwer** [comment]
I tracked a fix to the validator for this issue in #2869.
